apps/omg_child_chain/test/omg_child_chain/integration/happy_path_test.exs

```diff
@@ -31,13 +31,14 @@ defmodule OMG.ChildChain.Integration.HappyPathTest do
   require OMG.Utxo
 
   @moduletag :integration
+  @moduletag :child_chain
   # bumping the timeout to two minutes for the tests here, as they do a lot of transactions to Ethereum to test
   @moduletag timeout: 120_000
 
   @eth OMG.Eth.RootChain.eth_pseudo_address()
   @interval OMG.Eth.RootChain.get_child_block_interval() |> elem(1)
 
-  @tag fixtures: [:alice, :bob, :omg_child_chain, :token, :alice_deposits]
+  @tag fixtures: [:alice, :bob, :omg_child_chain, :token, :alice_deposits, :eth_node]
```

Comment:
are you sure this is required and isn't a leftover of some kind? `:eth_node` should be already stood up when `:omg_child_chain` is. Let's maybe try removing the explicit fixture once more.

Reply:
I've looked at the :omg_child_chain fixture and I can't find anything that would start :eth_node... Where are you seeing it?

I've added :eth_node as one of the last commits, there were runs that failed before. For example: https://circleci.com/gh/omisego/elixir-omg/5416

Reply:
it is started by it's dependency fixtures `:omg_child_chain` -> `:root_chain_config` -> `:contract` IIRC.

The error you link to... :thinking: hmm strange, why would this happen? Anyway, this looks as unavailability of the child chain server rather than of the eth node. And apparently it does fail for the second tx sent.

Reply:
In my opinion it’s because of pub/sub alarms that put the rpc in 503 mode. 
EthereumClientMonitor eventually raises and alarm that the client isn’t reachable and that puts the API offline.

Reply:
If we look at the `warn` logs in the failing test above we can see that:
```
  * test deposit, spend, restart, exit etc works fine2019-04-28 21:46:15.063 [warn] module=OMG.EthereumClientMonitor function=handle_event/2 ⋅Health check raised :ethereum_client_connection alarm.⋅
2019-04-28 21:46:15.063 [warn] module=OMG.RPC.Plugs.Health function=handle_event/2 ⋅Alarm ethereum_client_connection was raised.⋅
```

Reply:
but why would such explicit inclusion of the fixture affect this situation? or why would this become an issue in this particular PR? This doesn't fit together - the eth node should be up and running at this point of test, until the teardown code is executed. 

The contracts got deployed fine and deposits were made, so why would the ethereum node disappear suddenly, causing the alarm to fire?

Reply:
Looking into this further - my current take is that what happens is that there is a race condition between stopping the applications and EthereumClientMonitor.